app/lib/esClient.js

```diff
@@ -0,0 +1,80 @@
+const log = require('./logger');
+const elasticsearch = require('elasticsearch');
+const esConfig = require('../../config/config').es;
+
+const client = elasticsearch.Client({ host: `${esConfig.host}:${esConfig.port}` });
+
+function getBaseQuery(size) {
+  return {
+    index: esConfig.index,
+    type: 'pharmacy',
+    body: {
+      size,
+      query: {
+        bool: {}
+      }
+    }
+  };
+}
+
+function build(location, radius, size) {
+  const query = getBaseQuery(size);
+
+  query.body.query.bool.filter = {
+    geo_distance: {
+      distance: `${radius}mi`,
+      'location.coordinates': {
+        lon: location.longitude,
+        lat: location.latitude
+      }
+    }
+  };
+
+  query.body.sort = [
+    {
+      _geo_distance: {
+        'location.coordinates': {
+          lon: location.longitude,
+          lat: location.latitude
+        },
+        order: 'asc',
+        unit: 'mi',
+        distance_type: 'plane'
```

Comment:
I wonder if this should be `arc` given the relatively high latitudes some of the UK is at?
https://www.elastic.co/guide/en/elasticsearch/reference/5.4/query-dsl-geo-distance-query.html#_options_4

Reply:
There's a trade off between plane which is fast, and arc which is slower but more precise.
 there is some discussion in this thread https://discuss.elastic.co/t/internal-geo-distance-calculation-in-elastic-search/15889/3.

The mongoDB search used a spherical calculation rather than a planar one, so to maintain consistency I think it should change to `arc`.

Reply:
It would be interesting to see what the performance differences are. I'd much prefer to use `arc` for the accuracy and as you say, to maintain the behaviour we had with Mongo.